rust/Cargo.toml

```
@@ -7,7 +7,7 @@ categories = [ "science::robotics", "compression" ]
 repository = "https://github.com/foxglove/mcap"
 documentation = "https://mcap.dev/docs/rust/doc/mcap/index.html"
 readme = "README.md"
-version = "0.3.0"
+version = "0.3.1"
```

```suggestion
version = "0.4.0"
```
This isn't a bug fix, may as well bump the minor version.